modules/deploying-serverless-apps.adoc

```diff
@@ -19,3 +19,8 @@ $ oc apply --filename service.yaml
 Now that service has been created and the application has been deployed, Knative will create a new immutable revision for this version of the application.
 
 Knative will also perform network programming to create a route, ingress, service, and load balancer for your application, and will automatically scale your pods up and down based on traffic, including inactive pods.
+
+[NOTE]
+====
+The first time that a Knative service is created in a namespace, that namespace will require a one-time networking setup. This may cause the initial service to take longer than is usually required for a service to become ready.
```

Comment:
Can you be more specific? What networking setup? It would be also nice to describe how an upgrade from a previous version should be performed and what the user should expect. Please ignore this if you plan to describe it somewhere else or there's no difference between the upgrade and fresh new install.

Reply:
The one time upgrade will be described in the release notes I think. Other than that there should not be user-intervention necessary for updates.

Reply:
OK. Cool. Then it would make sense to rephrase it a little bit so that users don't think they "should" do something. Perhaps something like:
... that namespace will require a one-time networking setup. -> ...that namespace will automatically receive a new networking configuration.

Reply:
Yes, it's just that. Please refer to https://jira.coreos.com/browse/SRVKS-285 for the upgrade procedure that we want to do.

Reply:
Thanks @markusthoemmes - note is updated now.
I will add a new PR to deal with any upgrading docs separately, and the release notes are being covered under SRVCOM-579.